bin/install-qa-check.d/20perl-directories

```diff
@@ -0,0 +1,22 @@
+# Check for perl modules installed in invalid directories.
+
+perl_dir_check() {
+	# See, for example, bug #876757
+	local dir f=()
+	for x in usr/share/perl5 ; do
```

Comment:
I suppose you want to check for `/usr/lib64/perl5/5.*` directory (i.e. stuff being installed outside `vendor_perl`) by packages other than perl itself?